### ISSUE TITLE
Crystal 0.34.0 compatibility

### ISSUE THREAD

**Issue by anamba** [jgaskins/neo4j.cr#7]
This should be an easy one, I only found one lone `Errno` in there (`src/neo4j/bolt/connection.cr:538`) that I think can be replaced with `IO::Error`.
https://github.com/crystal-lang/crystal/pull/8885

**Comment by anamba** [comment]
Oh, if you want to be fancy and support multiple Crystal versions there's a `compare_versions` macro. I usually look at bcardiff's PRs on other projects to figure out what I need to do, here's one example: https://github.com/stefanwille/crystal-redis/compare/master...bcardiff:crystal/0.34.0

**Comment by jgaskins** [comment]
Oh, nice! Sure, I'll give that a shot.

**Comment by jgaskins** [comment]
Sorry for the delay, I meant to get to this last weekend, but I just released v0.6.0 which includes cluster support and Crystal 0.34 compatibility.

The cluster support may be something you want to look into for `neo4j-model` since I may be moving toward the `Connection` being an implementation detail, not intended to be exposed. Instead, it'll be more like the Java driver where the connection is an implementation detail. I just pushed an update to the README with an example.
